src/components/debug-panel/debug-panel.tsx

```diff
@@ -159,6 +204,62 @@ export const DebugPanel = ({ onClose }: DebugPanelProps) => {
             }
           />
         </ItemContainer>
+        {debugOptions.showUVDebugTexture && (
+          <TextureControlPanel>
+            <PickPanePanel
+              pickPaneSetName={PickPaneSetName.uvDebugTexture}
+              group={TEXTURE_GROUP_PREDEFINED}
+              panelLength={5}
+              imageWidth={TEXTURE_ICON_SIZE}
+              imageHeight={TEXTURE_ICON_SIZE}
+            />
+            <PickPanePanel
+              pickPaneSetName={PickPaneSetName.uvDebugTexture}
+              group={TEXTURE_GROUP_CUSTOM}
+              panelLength={5}
+              imageWidth={TEXTURE_ICON_SIZE}
+              imageHeight={TEXTURE_ICON_SIZE}
+            />
```

Comment:
Why they devided in 2 groups? It is one list of images

Reply:
Removed the second